db/db.js

```diff
@@ -383,6 +383,15 @@ async function isOverdoseSuspected(xethru, session, location) {
     let condition2 = 1 * (session.still_counter > still_threshold); //seconds
     let condition3 = 1 * (sessionDuration > sessionDuration_threshold);
     let condition4 = 1 * (0);
+
+    if(condition2 == 1){
+      let alertReason = 'Duration';
+    }
+
+    if( (condition1==1) || (condition3==1) ){
```

Comment:
when you use 'let' inside these blocks, you're only declaring a local variable - so unless the code that's using them is inside what gets executed by the if statement, alertReason will be undefined, and the update od_flag query will fail.

Reply:
Ahh that makes sense. Thanks!